sdk/cognitivelanguage/azure-ai-language-conversations/samples/reslotion_and_pii/sample_conv_pii_transcript_input.py

```diff
@@ -0,0 +1,134 @@
+    endpoint = os.environ["AZURE_CONVERSATIONS_ENDPOINT"]
+    key = os.environ["AZURE_CONVERSATIONS_KEY"]
+
+    # analyze quey
+    client = ConversationAnalysisClient(endpoint, AzureKeyCredential(key))
+    with client:
+
+        poller = client.begin_submit_conversation_job(
+            body=AnalyzeConversationJobsInput(
+                analysis_input=MultiLanguageConversationAnalysisInput(
+                    conversations=[
+                        TranscriptConversation(
+                            id=1,
+                            language="en",
+                            conversation_items=[
+                                TranscriptConversationItem(
+                                    id=1,
+                                    participant_id=0,
+                                    content=TranscriptConversationItemContent(
+                                        text="It is john doe.",
+                                        lexical="It is john doe",
+                                        itn="It is john doe",
+                                        masked_itn="It is john doe"
+                                    )
+                                ),
+                                TranscriptConversationItem(
+                                    id=2,
+                                    participant_id=1,
+                                    content=TranscriptConversationItemContent(
+                                        text="Yes, 633-27-8199 is my phone",
+                                        lexical="yes six three three two seven eight one nine nine is my phone",
+                                        itn="yes 633278199 is my phone",
+                                        masked_itn="yes 633278199 is my phone"
+                                    )
+                                ),
+                                TranscriptConversationItem(
+                                    id=3,
+                                    participant_id=1,
+                                    content=TranscriptConversationItemContent(
+                                        text="j.doe@yahoo.com is my email",
+                                        lexical="j dot doe at yahoo dot com is my email",
+                                        masked_itn="j.doe@yahoo.com is my email",
+                                        itn="j.doe@yahoo.com is my email"
+                                    )
+                                )
+                            ]
+                        )
+                    ]
+                ),
+                tasks=[
+                    AnalyzeConversationPIITask(
+                        parameters=ConversationPIITaskParameters(
+                            pii_categories=["all"]
+                        )
+                    )
+                ]
+            )
+        )
+
+        # view result
+        result = poller.result()
+        task_result = result.tasks.items[0]
```

Comment:
Should the sample loop over all the task items?

Reply:
yes! but in our case we submitted just one conversation item